src/index.ts

```diff
@@ -1,9 +1,9 @@
 import App from "./app";
-import * as env from "env-var";
 
-const PORT: number = env.get("PORT").default("5000").asPortNumber();
+// was : number specifically
+const PORT = process.env.PORT || 5000 //.asPortNumber();
```

Comment:
Can this be changed back to `const PORT: number = env.get("PORT").default("5000").asPortNumber();`

Reply:
Any particular reason why?